src/locale/translations/en.json

```diff
@@ -509,16 +509,15 @@
     },
     "CheckInView": {
       "Title": "Scanned",
-      "CTA1": "Ok",
+      "CTA1": "OK",
       "CTA2": "Cancel",
       "MoreInfo": "Learn more about QR codes and COVID alert"
     },
     "Exposed": {
       "Title": "You've been Exposed"
     },
     "CancelCheckIn": {
-      "Title": "Successfully cancelled check in",
-      "Body": "You have successfully cancelled this check in, you may close this screen"
+      "Title": "Scan cancelled"
```

Comment:
Do we have the FR string?

Reply:
Not in figma. I'll put in a placeholder.